### PR TITLE
[bitnami/opencart] Add VIB tests

### PR DESCRIPTION
**Pull request by marcosbc**
### Description of the change

The main objective of this PR is to publish our Bitnami OpenCart container using VMware Image Builder. In order to do that, several changes are included:

* Increasing the existing test coverage of the asset by adding Goss tests.
* Update verify and publish VIB pipeline's definitions.

### Benefits

* Ensuring higher quality of the container catalog.
* Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

N/A

### CODE REVIEW

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Could you please take a look at my comments?

File: .vib/opencart/goss/vars.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - /opt/bitnami/opencart/config.php
+      - /opt/bitnami/opencart/admin/config.php
+phpmodules:
+  # Based on https://docs.opencart.com/requirements/
+  - curl
+  - gd
+  - mbstring
+  - mysqli
+  - zip
+  - zlib
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/opencart/goss/vars.yaml:24]: Well, those are the requirements according to the official documentation. In the case of required PHP extensions, I would add those even if we are not explicitly adding them in the `postunpack` logic. If not, we would not be able to keep up with the references.

File: .vib/opencart/goss/opencart.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+command:
```
[the remark this answers is not in the document]
Comment (marcosbc) [reply .vib/opencart/goss/opencart.yaml:1]: The `upload` folder is the root directory for all OpenCart files. That is why it is not present in the image. We simply move all those files to `/opt/bitnami/opencart`. There isn't much we can check there.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Looks good to me, could you please remove the trigger?

**Comment by marcosbc** [comment]
Done @migruiz4!

**Review by migruiz4 — APPROVED** [review]

**Pull request merged**